uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/absence-tab/components/annual-entitlements.component.spec.js

```diff
@@ -118,6 +119,17 @@ define([
       });
     });
 
+    describe('when user wants to see a comment to an entitlement', function () {
```

Comment:
As per discussion with @AkA84 we decided just to change the colour of icon which will also change notification.error call to notification.info